### PR TITLE
CO-1501 implement get games by

### PR DESCRIPTION
**Pull request by Ruefa**
Implemented GET /games/{gameId}

### CODE REVIEW

File: api/v1/db/oracledb/games-dao.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ */
+const getGameById = async (id) => {
+  const connection = await conn.getConnection();
+  try {
+    const sqlParams = {
+      gameId: id,
+    };
+    const sqlQuery = `
+      SELECT ID AS "id", DEVELOPER_ID AS "gameId", NAME AS "name",
+      SCORE AS "score", RELEASE_DATE AS "releaseDate"
```
Comment (tsoliangwu0130) [inline api/v1/db/oracledb/games-dao.js]: The releaseDate format should follow the format you defined in openapi file here: https://github.com/osu-mist/game-curator-api/blob/master/openapi.yaml#L510. What I got currently is in this format: `"releaseDate": "28-OCT-08"`. You should make sure the raw data is serialized in your serializer.

File: api/v1/db/oracledb/games-dao.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+ *                            term is not found
+ */
+const getGameById = async (id) => {
+  const connection = await conn.getConnection();
+  try {
+    const sqlParams = {
+      gameId: id,
+    };
+    const sqlQuery = `
+      SELECT ID AS "id", DEVELOPER_ID AS "gameId", NAME AS "name",
```
Comment (tsoliangwu0130) [inline api/v1/db/oracledb/games-dao.js]: Can you break the line for each selected column as here to improve the readability? Example: https://github.sig.oregonstate.edu/ecs-data/students-api-contrib/blob/master/contrib.js#L11-L21

Also, `DEVELOPER_ID AS "gameId"` should be `developerId`.

File: api/v1/db/oracledb/games-dao.js (hunk 1 above)
Comment (tsoliangwu0130) [reply api/v1/db/oracledb/games-dao.js]: In addition, I tried to search the game with a score `GET /games/43`:

```json
{
    "links": {
        "self": "https://localhost/v1/games/43"
    },
    "data": {
        "type": "game",
        "id": "43",
        "links": {
            "self": "https://localhost/v1/games/43"
        },
        "attributes": {
            "name": "Path of Exile",
            "score": "2",
            "releaseDate": "15-OCT-13"
        }
    }
}
```

The score was parsed as string, but it should be a floating number: https://github.com/osu-mist/game-curator-api/blob/master/openapi.yaml#L501-L505

**Review by tsoliangwu0130 — CHANGES_REQUESTED** [review]

File: api/v1/db/oracledb/games-dao.js (hunk 1 above)
Comment (Ruefa) [reply api/v1/db/oracledb/games-dao.js]: Converted score and releaseDate into the correct formats.

File: api/v1/db/oracledb/games-dao.js (hunk 2 above)
Comment (Ruefa) [reply api/v1/db/oracledb/games-dao.js]: Improved readability and fixed miss named alias

**Review by tsoliangwu0130 — APPROVED** [review]

**Pull request merged**